src/config/constants.js

```diff
@@ -3,3 +3,7 @@ export const MAIN_URL = 'https://pfa.foreca.com/api/v1';
 export const minSearchCharacters = 3;
 export const EMAIL_REGEXP = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
 export const minStarLength = [...Array.from({ length: 5 }, (_, i) => i + 1)];
+
+// Reducer types
+export const SAVE_SEARCH = 'SAVE_SEARCH';
```

Comment:
please move action names to more specific file or rename so it would be clear what way the variables should be used

Reply:
I created new file `actionTypes.js` and moved variables here.